src/main/java/mops/module/wrapper/ModulWrapper.java

```diff
@@ -0,0 +1,20 @@
+package mops.module.wrapper;
+
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import mops.module.database.Modul;
+import mops.module.database.Veranstaltung;
+import mops.module.database.Veranstaltungsform;
+import mops.module.database.Zusatzfeld;
+
+@Data
+@AllArgsConstructor
+public class ModulWrapper {
+
+    Modul modul;
+    public List<Veranstaltung> veranstaltungen;
+    public List<Veranstaltungsform> [] veranstaltungsformen;
+    public List<Zusatzfeld> [] zusatzfelder;
```

Comment:
(siehe Anmerkungen im ModulWrapperService)
```suggestion
    public List<List<Veranstaltungsform>> veranstaltungsformen;
    public List<List<Zusatzfeld>> zusatzfelder;
```